viewer/packages/tools/src/Geomap/Geomap.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as GEOTHREE from 'geo-three';
+import { Cognite3DViewer } from '@reveal/core';
+import { APIKeys } from './MapConfig';
```

Comment:
This file seems to have 4 spaces as tabs, the other files have 2 spaces as tabs.